docs/changelog/4_x.rst

```diff
@@ -64,18 +66,18 @@ dump-and-restore process.
 
    .. code-block:: bash
 
-       $ sudo apt-get update && sudo apt-get install edgedb-4-beta1
+       $ sudo apt-get update && sudo apt-get install edgedb-4
 
    Under CentOS/RHEL, use this installation command:
 
    .. code-block:: bash
 
-       $ sudo yum install edgedb-3
+       $ sudo yum install edgedb-4
 
    In any required ``systemctl`` commands, replace ``edgedb-server-3`` with
    ``edgedb-server-4``.
```

Comment:
Almost deleted this, but I'm not sure. Is it still relevant post-release?

Reply:
Yes